spec/like_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.describe Like, type: :model do
+  before(:example) do
+    @user = User.new(name: 'Lilly', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Poland.')
+    @post = Post.new(author: @user, title: 'Hello', text: 'This is my first post')
+  end
+
+  subject { Like.new(user: @user, post: @post) }
+
+  it 'name should be present' do
+    subject.user = nil
+    expect(subject).to_not be_valid
+  end
+
```

Comment:
- [ ] Also, it would be better if we write tests for the `update_likes_counter` method
![Screenshot from 2023-10-09 08-45-50](https://github.com/mahdinoori2000/blog-app/assets/46388113/e81fc579-16f6-4bbb-91f2-8f5bb3fbba50)